tools/export_model_images.py

```diff
@@ -24,26 +29,22 @@
     "slack",
     "vsphere",
 ]
-required_providers = {"aws", "digitalocean", "gcp", "github", "kubernetes", "onelogin", "onprem", "slack", "vsphere"}
```

Comment:
This should be restored. It is a list of providers that exist in all documented versions on resoto.com.